.jshintrc

```diff
@@ -1 +1 @@
-./node_modules/jsgreat/.jshintrc
```

Comment:
Let's make this an `extends` instead of a symlink. I think we want to add `module: true` too. You'll need a second .jshintrc referencing lint-review's copy of jsgreat.